packages/ui/components/dialog/Dialog.tsx

```diff
@@ -153,7 +153,7 @@ export function DialogHeader(props: DialogHeaderProps) {
 
 export function DialogFooter(props: { children: ReactNode; className?: string; showDivider?: boolean }) {
   return (
-    <div className={classNames("bg-default sticky bottom-0", props.className)}>
+    <div className={classNames("bg-default sticky", props.className)}>
```

Comment:
add bottom-0 like i said in previous comment

Reply:
Updated the changes

Reply:
Added the bottom-0